### DISCUSSION TITLE
Orchard Harvest 2024 conference in Las Vegas!

### DISCUSSION THREAD

**Discussion by viktoriamagyar** [OrchardCMS/OrchardCore#15260] [General]
After last year, the Orchard Harvest Conference will be held again in 2024. Last year it was held online due to economic reasons. This year we would like to try again to organise it in person. But first, we would like to assess the potential interest and what would be needed.
You can fill in the questionnaire here: https://forms.office.com/e/XFpdexc2VU
It should take about 5 minutes.
Share your thoughts with us in the survey. We are curious about who we will meet in 2024! :)

**Comment by Piedone** [comment]
Sign up to receive our announcements about Orchard Harvest [here](http://eepurl.com/iKy3a2)! You can unsubscribe anytime.

**Comment by viktoriamagyar** [comment]
If you haven't filled out our questionnaire you have still time for it this week. :) Share your thoughts with us!

**Comment by viktoriamagyar** [comment]
Hello @OrchardCMS 
Based on feedback and other important factors, the organizing team has decided to finally organize the Orchard Harvest 2024 Conference in person this year. Each year we try to attract as many people as possible, and a face-to-face event helps to build truly productive relationships.
Last year, we started organizing the event in Las Vegas, so that's one of the reasons we're going to do it here again this year! 
Also most of the active people in the community are either from the US, or can reach the US.
I will be able to give you an exact date soon. What is certain is that it will take place in the autumn. Most likely in September.

**Comment by viktoriamagyar** [comment]
Hello @OrchardCMS
According to the questionnaire, most people wanted the event to take place in September. With the organizers, we agreed that a Thursday and Friday would be best, so it wouldn't take so many working days away from people. They could even stay for the weekend on an individual basis.
Finally, we chose the 19th and 20th of September. It's not right after the summer holidays, people don't travel as much then. There are fewer things that could come up for potential participants that would prevent them from participating. Also important is that it does not interfere with any national holiday.

Please mark it in your calendar:
![image](https://github.com/OrchardCMS/OrchardCore/assets/84029837/9620c00b-5376-4d24-bd02-564b8ca5d8bd)

**Comment by viktoriamagyar** [comment]
@OrchardCMS I didn't expect to have to change my announcement so quickly. Unfortunately, due to the capacity of the venue, we had to bring the event forward by a week. Unless this is influenced by external factors, this will be the fixed date!
Please mark it in your calendar:
**Orchard Harvest Conference 12th and 13th of September, Las Vegas**

**Comment by larremp** [comment]
As for a goal for 2024 Orchard Harvest, I would like to suggest a team building activity, so we can further deepen our personal relationships. Here are some possible activities we can do as a group, and maybe we can just select one for this year:
1. Watch a show at The Sphere. Maybe do this in one of the evenings. https://www.thespherevegas.com/
2. Visit Zion National Park (https://www.nps.gov/zion/index.htm). Maybe we can do this on the Saturday. Leave early morning, drive (around 2 hours), explore and drive back in the afternoon.

Maybe @malhayek2014 knows other possible outdoor activities we can do. I would think September weather would be nice and NOT too hot.

**Reply by MikeAlhayek** [reply]
@larremp I did not now @malhayek2014 is a username until you used it :).
 Vegas is becoming a sport town, we can watch Hockey, Baseball, Basketball, American Football. Not sure what will be available in September other than Hockey preseason. 

But, also, not sure that everyone will be okay with attending this. We can throw some ideas out there and see if there is groups that would be interested or have the time.

**Reply by Piedone** [reply]
Behold, an impostor!

We could definitely have extra programs for those interested, great idea!

**Reply by viktoriamagyar** [reply]
Having a rather personal team building is a really nice idea! :)

**Comment by viktoriamagyar** [comment]
@OrchardCMS We are currently working with the organisers on a contract with the venue. We are also thinking about what themes the event should be based on.  Furthermore, we are trying to put together a set of goals that we will try to follow through.
Do you have any ideas that you would like us to consider?

**Reply by Piedone** [reply]
My take on themes: Leading up to a v2.0 (https://github.com/OrchardCMS/OrchardCore/discussions/15530), how will the future OC look like? What's the role of a CMS nowadays (with the decoupled/headless operating models, AI development tools)?

Goals:

- Meet in-person finally again with other community members (the last time was 2017).
- Welcome newly active members.

BTW tagging OrchardCMS doesn't notify anybody.

**Comment by viktoriamagyar** [comment]
I would like to note that there is a possibility of sponsoring the Orchard Harvest conference, and we are currently in the planning stage. We would be open to it if you would like to sponsor our event. If you are interested, you can contact us using [orchardharvest@lombiq.com](mailto:orchardharvest@lombiq.com) or the [contact@orchardcore.net](mailto:contact@orchardcore.net) email addresses, and you can also write us here.
If you would like to be kept informed about the events around the conference [sign up here](http://eepurl.com/iKy3a2) to receive our announcements about Orchard Harvest.

**Comment by viktoriamagyar** [comment]
As we continue to organize the conference, we have started to look in more detail at the photography and video of the event. This is very important to us as we know that unfortunately, not everyone can attend the event, but we want to be able to create value for everyone in the community. However, hiring a professional team would be a huge expense for us so we are using our collective resources and knowledge to ensure quality recording.  If anyone has any ideas or tools to offer we would welcome them.
Also, @MikeAlhayek will be visiting our chosen location this week to make sure all our needs are being met. If all goes well, we should be able to officially announce the location in Las Vegas soon :)

**Comment by viktoriamagyar** [comment]
I am happy to announce that the Orchard Harvest Conference will take place at [The Orleans Hotel & Casino](https://g.co/kgs/L3U3eBo)!
It will be a truly unique and real Las Vegas experience for all participants.
Registration and hotel reservation details will be available soon, as the hotel will also offer an extra discount for those who wish to stay there during the conference.

**Comment by viktoriamagyar** [comment]
Hi Orchard Community members,

Get ready to power up your Orchard skills at Orchard Harvest Conference 2024!  Join us on September 12th-13th at the Orleans Hotel and Casino in lively Las Vegas for two days packed with learning, coding, and community fun.

What can you expect at Orchard Harvest Conference 2024?
- **Hands-On Workshops:** Elevate your coding game with interactive sessions led by industry experts.
- **Insider Insights:** Learn best practices, advanced techniques, and real-world insights from qualified developers.
- **Global Connections:** Connect with fellow enthusiasts, exchange ideas, and forge meaningful relationships within the Orchard community.
- **Special Perks:** Participants get exclusive discounts on accommodations at the Orleans Hotel and Casino.

Can't wait until September? Check out recordings from last year's special online Orchard Harvest on our YouTube channel [here](https://youtube.com/playlist?list=PLpCsCyd254Fr-2gbJEOLAJaoU_4EqHKaI&si=FUxRXPdix9F-xDIZ).

Ready to be a part of something extraordinary? Reserve your spot today and take advantage of early-bird pricing at [Orchard Harvest Conference 2024](https://us.commitchange.com/nv/las-vegas/orchardcore/events/orchard-harvest-2024) 

Want to support our mission? Become a sponsor! Reach out to us at [orchardharvest@lombiq.com](mailto:orchardharvest@lombiq.com) or [contact@orchardcore.net](mailto:contact@orchardcore.net) to explore sponsorship opportunities.

Secure your spot today and get ready to level up your skills at Orchard Harvest Conference 2024!
See you there!

**Comment by viktoriamagyar** [comment]
We also opened the registration form for the speakers:
https://forms.office.com/e/fewh7hh20d

This year's leading themes:

- Leading up to a v3.0, how will the future OC look like? 
- What's the role of a CMS nowadays (with the decoupled/headless operating models, AI development tools)
- Explore some advanced topics such as Shapes, Placements, Cloud Integrations, Performance, module extensibility, etc.
- Showcases on implementing OC in action.

Apply to be a speaker until the **15th of June, midnight Anywhere on Earth!** We'll notify you whether your talk is selected for Harvest until the end of June. The chosen speakers will receive complimentary tickets for the event.

**Comment by viktoriamagyar** [comment]
Hi Orchard Community Members!

Please if you see our posts on any social media platforms, share them with your friends and colleagues, so we can reach as many people as possible.
Also, just a quick reminder that you can still apply to be a speaker at the conference! https://forms.office.com/e/fewh7hh20d
Thank you! :)

**Comment by viktoriamagyar** [comment]
**Harvest 2024 - Call for Speakers!**

Are you passionate about technology and innovation? Do you have insights on the future of OC, CMS roles, or advanced tech topics? Harvest 2024 is looking for dynamic speakers to share their expertise with our community. Apply now to be a part of this transformative event!

**Register Here:** https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u

**This Year's Leading Themes:**
Leading Up to a v3.0: The Future of OC
- How will the future OC look like?
The Role of a CMS nowadays
- Decoupled/headless operating models
- AI development tools
Explore some advanced Topics
- Shapes
- Placements
- Cloud Integrations
- Performance
- Module Extensibility
Showcases
- Real-world implementations of OC in action.

**Important Dates:**
Speaker Application Deadline: June 15th, Midnight (Anywhere on Earth)
Notification of Selection: By the end of June
Benefits for Chosen Speakers:
Complimentary tickets for the event

We welcome speakers of varying experience levels, don't miss this opportunity to connect with like-minded professionals. Apply to be a speaker at Harvest 2024 today!

You can find all the important details at https://orchardcore.net/harvest
We look forward to seeing your applications and hearing your innovative ideas!

**Comment by viktoriamagyar** [comment]
Calling all Orchard Community tech enthusiasts! Apply to speak at Harvest 2024 and share your insights on OC's future, CMS roles, AI tools, cloud integrations, and more!

Regardless of your experience level, and whether you have a major presentation or a smaller talk, we want to hear your insights! Your unique perspective is valuable to us. Apply by June 15, midnight AoE.Chosen speakers get free event tickets! 👉 [Speaker Registration Form](https://forms.office.com/e/fewh7hh20d)

Leading Themes:

Leading Up to a v3.0: The Future of OC
The Role of a CMS in the Modern World
Advanced Topics: Shapes, Placements, Cloud Integrations, Performance, Module Extensibility
Showcases: Real-world implementations of OC in action
Important Dates:

Speaker Application Deadline: June 15th, Midnight (Anywhere on Earth)
Notification of Selection: By the end of June
Join us and be part of this transformative event!

**Reply by Piedone** [reply]
I've submitted two talks just now! I also urge you to take part and sign up for it, as well as submit a talk: https://orchardcore.net/harvest. Talks of all kinds and experience levels are welcome, you don't need to be a hardcore Orchard Core professional, just have something you're excited to share!

**Reply by viktoriamagyar** [reply]
@ns8482e, @MikeKry, @microposmp, @infofromca, @MarGraz, @brunoAltinet 
I couldn't contact you privately, but I would like to invite you to be a speaker at Orchard Harvest 2024 in Las Vegas!
We at the organizer team selected you, among a handful of other community members, to invite to give a talk due to your activities in the community.

**Reply by MikeKry** [reply]
@viktoriamagyar Thanks for reaching out! I would be glad to participate in Orchard Harvest. I need to check if it is possible for me and will let you know.

**Reply by viktoriamagyar** [reply]
Dear @MikeKry! 
Thank you, it would be really nice if you could join as well. Also, We would be very open to your presentation and would be grateful if you could contribute to the success of the event. :) You can register here: https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u

If you have any questions do not hesitate to contact us at orchardharvest@lombiq.com

**Reply by MarGraz** [reply]
@viktoriamagyar, thank you for your invitation. I have some questions that I will ask here because I think they might be useful for others as well:

1. Is it possible to participate in the talk remotely? For example I'm based in Europe;
2. What is the main topic of the talk?
3. Is it necessary to prepare something like slides, and should they be shared with you before?
4. If I'm part of a company, is it possible to display the company logos?

Thank you ☺️

**Reply by viktoriamagyar** [reply]
@MarGraz Hi! Thank you for the questions.

1. We mainly expect live performances. It would be great if you could join us live, giving you the opportunity to get to know the community.
2. This year's leading themes:

- Leading up to a v3.0, how will the future OC look like? 
- What's the role of a CMS nowadays (with the decoupled/headless operating models, AI development tools)
- Explore some advanced topics such as Shapes, Placements, Cloud Integrations, Performance, module extensibility, etc.
- Showcases on implementing OC in action.

3. Check out our toolbox to help you get going: https://github.com/OrchardCMS/Orchard-Harvest-Speaker-Toolbox.
4. Do you mean on the slides?

You can register here: https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u All the important details are in the description boc.

**Reply by MarGraz** [reply]
Hi @viktoriamagyar, 

thank you for your answer.
Unfortunately is not possible for me in September to join you in Las Vegas. Next time if you organize the harvest in Europe, I will try to participate.

Yes, about the question number 4, I meant on the slides.

Thank you

**Reply by viktoriamagyar** [reply]
I am sorry to hear that! I hope you can join us next time!

**Comment by viktoriamagyar** [comment]
Join us for Orchard Harvest Conference 2024 in vibrant Las Vegas on September 12th and 13th at the Orleans Hotel and Casino!
Get ready for two days filled with learning, coding, and connecting with the worldwide Orchard community.

During these two days, you'll have the chance to dive deep into Orchard Core development, learn best practices, and engage with other experts.
Can't wait until September? Check out recordings from last year's special online Orchard Harvest on our YouTube channel [here](https://youtube.com/playlist?list=PLpCsCyd254Fr-2gbJEOLAJaoU_4EqHKaI&si=ROuu49l38WM3QdKV).

But that's not all! As a participant, you'll enjoy exclusive access to discounted accommodations at the Orleans Hotel and Casino.

Harvest 2024 still has a few places for dynamic speakers as well to share their expertise with our community. Regardless of your experience level, and whether you have a major presentation or a smaller talk, we want to hear your insights! **We've extended the sign-up period by another 2 weeks**, so if you haven't signed up yet, you can still sign up now! **Apply to be a speaker until the 29th of June**, midnight Anywhere on Earth! We'll notify you whether your talk is selected for Harvest in 1 or 2 weeks after the application period. The chosen speakers will receive complimentary tickets for the event.
[Register here](https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u)!

This Year's Leading Themes:

1. Leading Up to a v3.0: The Future of OC
- How will the future OC look like?
2. The Role of a CMS nowadays
- Decoupled/headless operating models
- AI development tools
3. Explore some advanced Topics
- Shapes
- Placements
- Cloud Integrations
- Performance
- Module Extensibility
4. Showcases
- Real-world implementations of OC in action.

Ready to be a part of something extraordinary? Reserve your spot today and take advantage of early bird pricing.

Don't miss this opportunity to connect with like-minded professionals. 
You can find all the important details at [https://orchardcore.net/harvest](https://orchardcore.net/harvest)

**Reply by hishamco** [reply]
BTW is recorded videos accepted, for those who can attend, especially the community coz some of them are from Thailand, China, and other places that far away from USA

**Reply by MarGraz** [reply]
@hishamco I didn't understand if recorded video are accepted? 🤔

**Reply by hishamco** [reply]
I mean recorded session in case someone unable to attend

Or streaming via Teams as we said before

**Comment by viktoriamagyar** [comment]
### Speaker Application Deadline Extended for Harvest 2024!

Dear Orchard Community,

We have exciting news, you still have a chance until June 29th, midnight AoE, to apply and share your insights with the community.

We invite you to present on a variety of topics including the future of Orchard Core, the role of CMS in the modern world, advanced technical topics, and real-world implementations. Whether you’re an experienced speaker or new to presenting your unique perspective is valuable to us.
**Leading Themes:**
	•	Leading Up to a v3.0: The Future of Orchard Core
	•	The Role of a CMS in the Modern World
	•	Advanced Topics: Shapes, Placements, Cloud Integrations, Performance, Module Extensibility
	•	Showcases: Real-world implementations of Orchard Core in action

**Important Dates:**
	•	Extended Speaker Application Deadline: June 29th, Midnight (Anywhere on Earth)
	•	Notification of Selection: Within 1-2 weeks after the application period ends

Chosen speakers will receive free event tickets, so don’t miss this opportunity to contribute to the community and connect with fellow tech enthusiasts.
Apply here: [Speaker Registration Form](https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u)
We look forward to your submissions!

The Orchard Harvest 2024 Team

**Comment by viktoriamagyar** [comment]
🚨 Last call for speaker applications and early bird tickets for Harvest 2024! 🚨

This is your final call to share your insights on Orchard Core, CMS roles, AI tools, cloud integrations, and more. Submit your speaker application by June 29th, midnight AoE. Chosen speakers get free event tickets!

Key Themes:
* Leading Up to a v3.0: The Future of Orchard Core
* The Role of a CMS in the Modern World
* Advanced Topics: Shapes, Placements, Cloud Integrations, Performance, Module Extensibility
* Showcases: Real-world implementations of Orchard Core in action

🔗 [Apply to be a Speaker](https://forms.office.com/pages/responsepage.aspx?id=Wt6elek45kStyIVVO-uCICDdeeZFbXdAlSWTkdgpZAtUQTJHWEQzSkg4WjRVOEFGUkpBWFlFMTM4QS4u)

If you're looking to attend as a participant, early bird tickets are still available at a discounted rate. Don’t miss this opportunity to join a vibrant community of tech enthusiasts, learn from industry experts, and network with peers. Secure your spot at Harvest 2024 and enjoy insightful sessions, workshops, and more.

 🔗 [Get Early Bird Tickets](https://us.commitchange.com/nv/las-vegas/orchardcore/events/orchard-harvest-2024)

All the information you need can be found here: [Orchard Harvest 2024](https://orchardcore.net/harvest)

**Comment by viktoriamagyar** [comment]
### The full program of Orchard Harvest 2024 is finally here!
The full Orchard Harvest program has finally arrived. This year’s program is packed with insightful sessions, engaging panels, and ample opportunities to connect with the Orchard community. Here is a detailed schedule to help you prepare for the conference.
### First Day:
**Morning Sessions:**
* **8:00 AM - 9:00 AM:** Registration, Coffee, and Breakfast Snacks 
* **9:00 AM - 9:35 AM: Opening Session** 
Kick off the conference with a warm welcome and opening remarks.
* **9:45 AM - 10:00 AM: Announcement of the Winner of the Jean-Thierry Kéchichian Community Award** 
Join us in celebrating the work of an outstanding community member.
* **10:10 AM - 11:05 AM: To Be Announced** 
Stay tuned for an exciting session to be revealed soon. 
* **11:15 AM - 12:10 AM: Demystifying Shapes** with Sebastien Ros
 Gain deep insights into shapes in Orchard. 
* **12:10 PM - 1:40 PM: Lunch Break** 

**Afternoon Sessions:**

* **1:40 PM - 2:35 PM: Content Warning Module Demo by Drew Brasher** 
Discover in live action how The Content Warning module enables us to hide some content behind a warning. 
* **2:45 PM - 3:45 PM: Panel Discussion** 
Submit your questions in advance and engage with our expert panel. 
* **3:45 PM - 4:05 PM: Coffee Break and Snacks** 
* **4:05 PM - 4:40 PM: Using Orchard as a Central Portal for the Biggest Small Business Association in SE Europe** by Bruno Samardzic 
Study the ups and downs of using Orchard to implement a portal.  
* **4:50 PM - 5:45 PM: From Zero to Azure: Setting Up Your Orchard Application for Success** by Benedek Farkas
 Get practical tips for deploying your Orchard application on Azure. 
* **5:45 PM: Closing Session**
### **Evening:**
* **Social Event - Dinner**
Unwind and network with fellow attendees over dinner.

### Second Day:
**Morning Sessions:**
* **8:00 AM - 8:30 AM: Coffee and Breakfast Snacks**
* **8:30 AM - 9:00 AM: Building the Orchard Core Community** by Zoltán Lehóczky 
Let’s go through what has happened, what effects certain changes had on our community life, and a vision for what we’ll see next.
* **9:10 AM - 10:05 AM: Understanding the Performance Characteristics of Orchard Core** by Sebastien Ros 
Gain insights into optimizing Orchard Core performance. 
* **10:20 AM - 10:55 AM: Smithsonian Folkways Collection Management and Orchard Core Demonstration** by Toby Dodds 
See how Orchard Core is used in a real-world project. 
* **11:10 AM - 12:05 PM: Unlocking Hyperproductivity with Orchard, Elsa, and Intelligent Agents** by Sipke Schoorstra 
Explore the integration of Orchard Core, Elsa Workflows, and intelligent agents to revolutionize content management. 
* **12:05 PM - 1:35 PM: Lunch Break**

**Afternoon Sessions:**

* **1:35 PM - 2:10 PM: Security Scanning an Orchard Core Application** by Zoltán Lehóczky
 Learn about how you can scan your Orchard Core app for security issues with Zed Attack Proxy. 
* **2:20 PM - 3:00 PM: Speeding Up: Custom Development** by Michal Kužela
 Find out how to speed up custom development via modules that bring new possibilities to OrchardCore. 
* **3:10 PM - 5:10 PM: Hackathon** 
Collaborate and innovate with fellow attendees over coffee and snacks.
* **5:15 PM: Closing Session** 

Do not forget to purchase your ticket at an early bird price until the 15th of August: [here.](https://us.commitchange.com/nv/las-vegas/orchardcore/events/orchard-harvest-2024)

**Comment by Piedone** [comment]
The full program with all the speaker and talk details are now out too! https://orchardcore.net/harvest

**Comment by viktoriamagyar** [comment]
Hello @OrchardCMS/developers !

### About Orchard Harvest
If you haven't heard about it, Orchard Harvest is the annual gathering of the Orchard Core community, with entry-level and hardcore technical talks, a review of where we are, a panel, and a hackathon.
### Why should you participate?
* You can attend exciting presentations by experienced Orchard Core users and ask your questions live, as we have organized Q&A sessions within the program.
* You'll also have plenty of opportunities to meet and network with technology experts and frequent users, including a panel discussion on today's most trending topics.
* Make your .NET web developer team more productive with a standardized framework and CMS, and stop reinventing the wheel (we already have reinvented it :)). Harvest is the most engaging way to start with Orchard Core.
### Early Bird Offer
You can get tickets at an early bird discount until the 15th of August.
### Event Details
* Location: Las Vegas, The Orleans Hotel & Casino
*  Dates: 12th and 13th of September, 2024
* Early Bird Tickets: Available until the 15th of August

For more details and to register, please visit our [Orchard Harvest 2024](https://orchardcore.net/harvest) page.
We would love to meet fellow Orchard developers like you in person at Orchard Harvest 2024. 

Still hesitating about joining us? Do you perhaps have questions about the program that I can help with to ensure it's an event indeed useful for you?
Looking forward to welcoming you in Las Vegas!

**Reply by Piedone** [reply]
And now we have @MikeAlhayek's talk up too! https://orchardcore.net/harvest

**Comment by viktoriamagyar** [comment]
🚨 Heads up, everyone! 🚨 
Just 2 weeks left to grab your early bird tickets for Orchard Harvest 2024 in Las Vegas! 🎟️

This is your chance to attend the Orchard Core event of the year, happening this September. Don't miss out on:
- Learning the latest features and best practices.
- Joining panels about the future of Orchard Core.
- Meeting and networking with technology experts and fellow community members.

Take advantage of early bird pricing and enjoy savings before the offer expires on August 15th!

Get your tickets now and secure your spot: [Orchard Harvest 2024 ](https://orchardcore.net/harvest)
Looking forward to seeing you there! 🌟

**Comment by viktoriamagyar** [comment]
One of the speakers at the Orchard Harvest Conference will be Benedek Farkas. Co-Founder and Managing Director of [Lombiq Technologies](https://lombiq.com/), university guest lecturer since 2014.

His presentation is called From zero to Azure: Setting up your Orchard application for success.

"In this talk I'll go through a complete process of setting up an Azure environment tailored for Orchard, enhanced by various open-source components to aid developers and maintainers."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Benedek Farkas](https://github.com/user-attachments/assets/94737a74-fe8e-4f98-99d6-8392bc01c82a)

**Comment by viktoriamagyar** [comment]
**Panel discussion questions**
Don’t miss your chance to shape the conversation at the Orchard Harvest Conference 2024! Whether you're attending in person or not, you can still submit questions and topics for our upcoming panel discussion. Influence what our experts will discuss by getting involved here: https://forms.office.com/e/fyusqNqJuG

**Comment by viktoriamagyar** [comment]
Bruno Samardzic is set to be one of the speakers at the Orchard Harvest Conference.  He is a senior .NET developer and entrepreneur.

His presentation is called Using Orchard as a Central Portal for the Biggest Small Business Association in SE Europe.

"This talk is a showcase of how we used Orchard to implement this portal and what problems we encountered in the meanwhile. It's now been running for 2 years. What we feel worked, what could be better, and what hacks we resorted to."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Bruno Samardzic](https://github.com/user-attachments/assets/143c677e-45bb-47d4-b2c9-d99f3f22604a)

**Comment by Piedone** [comment]
The sponsors of Orchard Harvest are now public! These are the companies that support the Orchard Core community and the conference:

- CrestApps (https://crestapps.com/)
- Lombiq (https://lombiq.com/)
- MCCi (https://mccinnovations.com/)

**Comment by viktoriamagyar** [comment]
The Orchard Harvest Conference will feature @DrewBrasher as a speaker. Software Development Manager with 10 years of experience in Software Development.

Her presentation is called Content Warning Module Demo.

"The Content Warning module lets you hide some content behind a warning so that users can decide if they want to view it or not. In this session, you will get to see it in action and take a look at the code that makes it happen. We will see how this is done with a Shortcode as well as a Content Part that adds a wrapper to the parent Content Item."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Drew Brasher](https://github.com/user-attachments/assets/fd280e16-cf3a-4bdc-b93a-837df471e97f)

**Comment by viktoriamagyar** [comment]
At the Orchard Harvest Conference, one of the speakers will be @MikeAlhayek. Software engineer specializing in ASP.NET Core and Orchard Core.

His presentation is called Authentication in Orchard Core.

"Join us for "Authentication in Orchard Core" to explore secure authentication and authorization within the Orchard Core framework. Learn about Single-Factor and Two-Factor Authentication, integrating external providers, and implementing OpenID Connect for enhanced security and user experience. This session will equip developers and IT professionals with the knowledge to secure their Orchard Core applications effectively."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.

![Mike Alhayek](https://github.com/user-attachments/assets/3eac8a78-4b79-4ffe-a64e-a21984000988)

**Comment by viktoriamagyar** [comment]
Michal Kužela will be speaking at the Orchard Harvest Conference. Backend developer, Team & Tech lead in BiQ pux a.s.

His presentation is called Speeding Up: Custom Development.

"The talk's focus is on speeding up custom development via modules that bring new possibilities to Orchard Core. We have developed a module for creation & migration of DB tables, a module to fill these tables via CSV/Excel import and a module that shows them in dynamic tables with filters, sorters and more. All editable in admin, (almost) no-code required, so developers can focus on more complex tasks. Still a WIP though."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Michal Kužela](https://github.com/user-attachments/assets/7846b326-486d-4ff8-885b-b0a2f4987b54)

**Comment by viktoriamagyar** [comment]
The conference lineup includes @sfmskywalker  as one of the speakers. Senior Full Stack Software Engineer specializing in .NET and cloud computing.

His presentation is called Unlocking Hyperproductivity with Orchard, Elsa, and Intelligent Agents.

"In this session, we will explore the integration of Orchard Core, Elsa Workflows, and intelligent agents to revolutionize content management. You will learn how to harness the power of AI to automate and enhance various stages of content creation and publishing using workflows and agents, achieving unprecedented productivity."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Sipke Schoorstra](https://github.com/user-attachments/assets/04576939-d9ff-4038-9177-a0179e3574ec)

**Comment by viktoriamagyar** [comment]
Toby Dodds is scheduled to speak at the Orchard Harvest Conference. Director of Web and IT at the Smithsonian Institution overseeing the technical well-being of the Smithsonian Folkways website and digital collection.

His presentation is called Smithsonian Folkways Collection Management and Orchard Core Demonstration.

"With a mission to increase and diffuse knowledge, Smithsonian Folkways Recordings digitizes and distributes a large catalog of historic sound recordings. At the middle of our enterprise is an Orchard Core implementation that has been customized to support the needs of our collection and website. By blending the role of website CMS with a robust collections management platform, Folkways has achieved its goals of broadly distributing its collection via its website and through multiple 3rd party music services and platforms - reaching millions of listeners per year."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.

![Toby Dodds](https://github.com/user-attachments/assets/cfa6f773-3a19-485d-9c43-e29417a76856)

**Comment by viktoriamagyar** [comment]
As a speaker, @Piedone will participate in the Orchard Harvest Conference. Co-founder and managing director of Lombiq.

His presentation is called Security Scanning an Orchard Core Application

"As a web developer, you have tons of security aspects to keep in mind: preventing XSS and SQL injection, configuring HTTPS and HSTS, setting up the proper CSP headers… While you do need to understand these, you have help doing them right: security scanning! In this talk, I'll show you how you can scan your Orchard Core app for security issues with Zed Attack Proxy, also doing it automatically as part of UI testing."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.

![Zoltán Lehóczky](https://github.com/user-attachments/assets/86546910-48dc-4e10-9606-ecd9c8f9c732)

**Comment by viktoriamagyar** [comment]
@sebastienros will take the stage as a speaker at the Orchard Harvest Conference. Developer at [Microsoft](https://www.microsoft.com/), benevolent dictator of Orchard Core.

His presentation is called Demystifying Shapes.

"Shapes are the arcane magic of Orchard Core's theming. Or are they? While they are awesome, there's no magic, and by the end of this session, you'll fully understand them too!"

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.
![Sebastien Ros](https://github.com/user-attachments/assets/1baf39d0-5e05-49a6-920d-0b4c4a08ee9c)

**Comment by viktoriamagyar** [comment]
@sebastienros  will take the stage as a speaker at the Orchard Harvest Conference. Developer at [Microsoft](https://www.microsoft.com/), benevolent dictator of Orchard Core.

His second presentation is called Understanding the Performance Characteristics of Orchard Core.

"Why is Orchard Core slow when it's slow, why it's fast when it's fast, and how can you figure all of this out? From the performance guy of the ASP.NET Core team."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.

If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: https://orchardcore.net/harvest for registration and other important details.

![Sebastien Ros (2)](https://github.com/user-attachments/assets/c30e87a1-f11a-49b3-8c45-5354173a2b43)

**Comment by viktoriamagyar** [comment]
As a speaker, @Piedone will participate in the Orchard Harvest Conference. Co-founder and managing director of Lombiq.
His second talk is about Building the Orchard Core Community.

"Over the course of the last year or so, we've seen numerous improvements to how we function as a community. In this talk, we'll go through what has happened, what effects certain changes had on our community life, and a vision for what we'll see next."

The event will take place in Las Vegas at the Orleans Hotel & Casino from the 12th to the 13th of September.
If you'd like to attend the presentations but haven't bought your tickets yet, head over to the Orchard Harvest site: [https://orchardcore.net/harvest](https://orchardcore.net/harvest) for registration and other important details.

![Zoltán Lehóczky](https://github.com/user-attachments/assets/454de4e3-08bc-4c5e-9d5f-cc14f164fd33)

**Comment by viktoriamagyar** [comment]
### Panel discussion

Don’t miss your chance to shape the conversation at the Orchard Harvest Conference 2024! Whether you're attending in person or not, you can still submit questions and topics for our upcoming panel discussion. Influence what our experts will discuss by getting involved here: https://forms.office.com/e/fyusqNqJuG

In this section, we would like to discuss the most important and interesting topics that we don't have time for in busy business hours.

Submit your questions in advance or bring surprise questions and engage with our expert panel. @MikeAlhayek, @tobydodds and @burkeholland will be there to answer your questions. Moderated by @sebastienros .

**Comment by Piedone** [comment]
Thank you, everyone, for taking part, this was an awesome conference! Recordings are coming soon.

**Reply by hishamco** [reply]
Thank you Zoltan & Lombiq guys to make this event happened 🥇💐

**Reply by brunoAltinet** [reply]
Thanks Zoltan for everything!

**Reply by Piedone** [reply]
Most of the organizational work was done by @viktoriamagyar and @MikeAlhayek :).

**Reply by hishamco** [reply]
Right, thanks so much, both Viki & Majd, I appreciate your effort

BTW don't forget my T-Shirt :)

**Reply by DrewBrasher** [reply]
It was a really great conference! @viktoriamagyar and @MikeAlhayek did a great job and it was nice to meet some of the community on person!

**Reply by viktoriamagyar** [reply]
I am really happy that you had a great time! See you next year hopefully! :)

**Reply by SzymonSel** [reply]
Looking forward to next year 🤞
